File: TaskCore/Models/Member.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskCore.Models
{
  public  class Member
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Member_Id { get; set; }
        public string MemberName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: TaskProject/ClientApp/src/app/shared/member.ts

export class Member {
      Member_Id :number;
      MemberName :string;
     Email :string;
     Phone :string;
     Address :string;
     IsDeleted :boolean;

    constructor(_Member_Id: number,_MemberName:string,_Email:string, _Phone :string,_Address :string,_IsDeleted:boolean) {
        this.Member_Id = _Member_Id;
        this.MemberName = _MemberName;
        this.Email = _Email;
        this.Phone = _Phone;
        this.Address=_Address;
        this.IsDeleted=_IsDeleted;
}

}
